cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -18,6 +18,17 @@ use cranelift::prelude::{
 use std::collections::HashMap;
 use std::ops::RangeInclusive;
 
+/// Generates a Vec with `len` elements comprised of `options`
+fn arbitrary_vec<T: Clone>(
+    u: &mut Unstructured,
+    len: usize,
+    options: &[T],
+) -> arbitrary::Result<Vec<T>> {
+    (0..len)
+        .map(|_| u.choose(options).map(Clone::clone))
```

Comment:
I believe you can write this as:

```suggestion
        .map(|_| u.choose(options).cloned())
```